hack/deploy-cnv.sh

```diff
@@ -98,5 +98,10 @@ spec:
   BareMetalPlatform: true
 EOF
 
+if [[ "${KUBEVIRT_RELEASE}" =~ 0.34 ]]; then
+    echo "creating KubeVirt config map"
+    oc create -f "https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/kubevirt/v0.34.2/manifests/testing/kubevirt-config.yaml"
```

Comment:
I doubt something will change there, but maybe download from https://.../**$KUBEVIRT_RELEASE**/manifests/testing/kubevirt-config.yaml

Reply:
I think this is not even necessary as IIRC the hco operator takes care of this.

Reply:
[Supposedly so](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/66b7b001be5b03dde561b03e6db3dbd5872b8589/pkg/controller/hyperconverged/hyperconverged_controller.go#L821), but we have seen in the 4.6 lanes that all tests BeforeEach's have failed since this ConfigMap was missing.